guide-us-english.tex

```diff
@@ -375,7 +375,7 @@ \section{Connecting the PO/SM Cycles}
 
 \subsection{Team-Level Process}
 
-The \textbf{Team-Level Process} constitutes the first touch point between the Scrum Master and Product Owner Cycles, and is laid out clearly in the Scrum Guide. It is composed of three artifacts, five events, and three roles. The goals of the team level process are to:
+The \textbf{Team-Level Process} is Scrum as prescribed by the Scrum Guide. The goals of the team level process are to:
```

Comment:
And no mention of the touch point of the two cycles? You're removing information here. Disagree on this commit.